index.html

```diff
@@ -1110,6 +1110,30 @@
   <div class="well">
   <h4>Normalize/Equalize Audio</h4>
 
+  <!-- phase shift -->
+  <span data-toggle="modal" data-target="#phase_shift"><button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="bottom" title="Inverses the audio phase of the second channel">Flip phase shift</button></span>
+  <div id="phase_shift" class="modal fade" tabindex="-1" role="dialog">
+    <div class="modal-dialog modal-lg">
+      <div class="modal-content">
+        <div class="well">
+          <h3>Flip audio phase shift</h3>
+          <p><code>ffmpeg -i <i>input_file</i> -af pan="stereo| c0=c0| c1=-1*c1" <i>output_file</i></code></p>
```

Comment:
Then I would suggest my 2<sup>nd</sup> variant with spaces on both sides of `|`.